cgo_libc_static/README.md

```diff
@@ -111,26 +110,27 @@ fatal error: unexpected signal during runtime execution
 goroutine 3890 [syscall, locked to thread]:
 runtime.cgocall(0x44c7f0, 0xc82036a8d8, 0xc800000000)
     /usr/local/go/src/runtime/cgocall.go:120 +0x11b fp=0xc82036a888 sp=0xc82036a858
-os/user._Cfunc_mygetpwuid_r(0x0, 0xc8203a8390, 0x7f3c5c000a10, 0x400, 0xc8200e4058, 0x7f3c00000000)
```

Comment:
I wouldn't remove this, this is the symbol you'd actually find in the symbol table